File: backend/src/config/storage.js
const path = require('path');
const fs = require('fs');

const STORAGE_TYPE = process.env.STORAGE_TYPE || 'local';
const UPLOAD_DIR = process.env.UPLOAD_DIR || path.join(process.cwd(), 'uploads');

if (!fs.existsSync(UPLOAD_DIR)) {
  fs.mkdirSync(UPLOAD_DIR, { recursive: true });
}

module.exports = { STORAGE_TYPE, UPLOAD_DIR };

File: backend/src/config/storage.py
import os
from pathlib import Path

STORAGE_TYPE: str = os.environ.get("STORAGE_TYPE", "local")
UPLOAD_DIR: str = os.environ.get("UPLOAD_DIR", str(Path.cwd() / "uploads"))

Path(UPLOAD_DIR).mkdir(parents=True, exist_ok=True)
